apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,14 +1,14 @@
 ack_generate_info:
-  build_date: "2021-08-02T22:37:09Z"
+  build_date: "2021-08-03T01:38:05Z"
   build_hash: 8b9ae5dfab674f9d759bf6ccb1aef88bc91f2e4f
   go_version: go1.15.6 linux/amd64
   version: v0.8.0
```

Comment:
Should this be updated to v0.9.0?

Reply:
I haven't yet updated to `0.9.0` - not for this PR at least.